frame/support/Cargo.toml

```diff
@@ -73,3 +73,7 @@ no-metadata-docs = ["frame-support-procedural/no-metadata-docs"]
 # By default some types have documentation, `full-metadata-docs` allows to add documentation to
 # more types in the metadata.
 full-metadata-docs = ["scale-info/docs"]
+# Generate impl-trait for tuples with the given number of tuples. Will be needed as the number of
+# pallets in a runtime grows. Does increase the compile time!
+tuples-96 = []
+tuples-128 = []
```

Comment:
Feels like we need to expose these features more thoroughly in our docs, starting from doc module comments on the support crate, otherwise it wouldn't be easily discoverable.

Reply:
That's a great idea, will do.